addons/wp/functions/fnc_doArtillery.sqf

```diff
@@ -135,15 +145,15 @@ if (canFire _gun && {_caller call EFUNC(main,isAlive)}) then {
     if (EGVAR(main,debug_functions)) then {
         ["%1 Artillery strike complete: %2 fired %3 shots at %4m", side _gun, getText (configOf _gun >> "displayName"), _rounds, round (_gun distance2D _pos)] call EFUNC(main,debugLog);
     };
+};
 
-    // clean markers!
-    if (_mlist isNotEqualTo []) then {
-        [
-            {
-                {deleteMarker _x; true} count _this
-            }, _mList, 60
-        ] call CBA_fnc_waitAndExecute;
-    };
+// clean markers!
+if (_markerList isNotEqualTo []) then {
+    [
+        {
+            {deleteMarker _x; true} count _this
```

Comment:
Can we make a standard forEach out of this?

Reply:
Wilco